examples/text-classification/run_glue.py

```diff
@@ -369,12 +387,13 @@ def compute_metrics(p: EvalPrediction):
 
         for eval_dataset, task in zip(eval_datasets, tasks):
             eval_result = trainer.evaluate(eval_dataset=eval_dataset)
+            eval_result["eval_n_objs"] = len(eval_dataset)
```

Comment:
Same, not sure this adds anything (I had put them in finetune_trainer for backward compatibility reasons mostly).